fonts.css

```diff
@@ -0,0 +1,29 @@
+@font-face {
+    font-family: 'ptserif';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)